2224.minimum-number-of-operations-to-convert-time.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/two-furthest-houses-with-different-colors/
```

Comment:
URL of this problem should be "https://leetcode.com/problems/minimum-number-of-operations-to-convert-time/"